code/parachain/runtime/composable/src/xcmp.rs

```diff
@@ -195,7 +195,7 @@ impl pallet_xcm::Config for Runtime {
 	type Origin = Origin;
 	type Call = Call;
 
-	const VERSION_DISCOVERY_QUEUE_SIZE: u32 = 100;
+	const VERSION_DISCOVERY_QUEUE_SIZE: u32 = 100; // any reasoning behind this number?
```

Comment:
copy pasted from Acala and Statemine. That is how much QueryVersions we handles. Will reject if too many. In other place we accept request only from UMP and HRMP from Relay and from Parachains.

Reply:
we need our monitor, sre, bots or bridge team to start metric in prometheus size of our queues. likely other teams do kind of this. non urgent. but should be planned. IBC will be similar. cc